Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/Login.cs

```diff
@@ -52,12 +55,18 @@ public class Login : MonoBehaviour
         public Toggle RememberMeToggle;
         public ErrorPopup ErrorPopup;
         public List<Web3AuthButtonAndProvider> Web3AuthButtons;
-
+        
         private bool useWebPageWallet;
```

Comment:
Let's place it together with `_redirectToWallet`. Also we don't use `_` prefix for private fiels.

Reply:
Also there is no webPageWallet anymore.

Reply:
sure I'll make the renames but here's a link to naming conventions for C# by Microsoft where private fields should be preceded by an '_' https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

Reply:
I know that Unity has their own convention but C#'s convention should overwrite Unity's

Reply:
done

Reply:
> sure I'll make the renames but here's a link to naming conventions for C# by Microsoft where private fields should be preceded by an '_' https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

It's funny because Arshia, who is a big fan of Microsoft, was telling me the exact opposite :D